base/linalg/arnoldi.jl

```diff
@@ -28,7 +28,17 @@ The following keyword arguments are supported:
 | `:SI`   | eigenvalues of smallest imaginary part (nonsymmetric or complex `A` only)                                                 |
 | `:BE`   | compute half of the eigenvalues from each end of the spectrum, biased in favor of the high end. (real symmetric `A` only) |
 
-* `tol`: tolerance (``tol \\le 0.0`` defaults to `DLAMCH('EPS')`)
+* `tol`: parameter defining the relative tolerance for convergence of Ritz values (eigenvalue estimates).
+     A Ritz value ``θ`` is considered converged when its associated residual
+     is less than or equal to the product of `tol` and ``max(ɛ^{2/3}, |θ|)``,
+     where ``ɛ`` is machine epsilon as determined by the LAPACK routine
+     `_LAMCH('EPS')`.
```

Comment:
It's useful to clarify which definition of machine epsilon is used. There are two conventions which differ by a factor of two. Julia uses the larger and LAPACK the smaller.